--- a/collections_1.cpp
+++ b/collections_1.cpp
@@ -120,7 +120,6 @@
 #include <BOPDS_Interf.hxx>
 #include <StepGeom_CompositeCurveSegment.hxx>
 #include <AIS_GlobalStatus.hxx>
-#include <IVtk_Types.hxx>
 #include <IGESDimen_GeneralNote.hxx>
 #include <SelectMgr_Selection.hxx>
 #include <Message_Msg.hxx>
@@ -205,7 +204,6 @@
 #include <Geom2d_BoundedCurve.hxx>
 #include <StepShape_Face.hxx>
 #include <Font_SystemFont.hxx>
-#include <IVtk_Types.hxx>
 #include <StepGeom_SurfaceBoundary.hxx>
 #include <OpenGl_GlTypes.hxx>
 #include <Graphic3d_ShaderVariable.hxx>
@@ -272,7 +270,6 @@
 #include <HLRBRep_ShapeBounds.hxx>
 #include <StepShape_OrientedEdge.hxx>
 #include <Graphic3d_ZLayerId.hxx>
-#include <IVtk_Types.hxx>
 #include <Geom_BSplineCurve.hxx>
 #include <Units_Quantity.hxx>
 #include <BOPTools_ConnexityBlock.hxx>
